tests/config/config.d/custom_settings_prefixes.xml

```diff
@@ -1,4 +1,4 @@
 <clickhouse>
     <!-- Comma-separated list of prefixes for user-defined settings. -->
-    <custom_settings_prefixes>custom_</custom_settings_prefixes>
+    <custom_settings_prefixes>SQL_,custom_</custom_settings_prefixes>
```

Comment:
Sadly, we use our own mini-language (comma-separated values) inside XML.
The right way would be:

```
<custom_settings_prefixes>
    <prefix>SQL_</prefix>
    <prefix>custom_</prefix>
</custom_settings_prefixes>
```